frontend/components/Catalogs/CatalogRecipes.tsx

```diff
@@ -59,15 +54,15 @@ function CatalogRecipes() {
   });
 
   const handleSupprClick = (id?: string) => {
-    const index = catalogRecipes.findIndex((recipe: RecipeToSendType) => {
+    const index = catalogRecipes.findIndex((recipe: CatalogRecipeToSend) => {
       return recipe.id === id;
     });
     const recipeToSend = catalogRecipes[index];
     void deleteCatalogRecipe({ recipeToSend });
```

Comment:
It would be better to do also the change here: give only the recipe ID and not the recipe itself